src/xo-app/new-vm/index.js

```diff
@@ -922,10 +928,20 @@ export default class NewVm extends BaseComponent {
             onChange={this._getOnChange('coresPerSocket')}
             value={coresPerSocket}
           >
-            <option> Choose core(s) per socket </option>
-            {map(this._getCoresPerSocketPossibilities(),
-              coresPerSocket => <option value={coresPerSocket}>{`${CPUs / coresPerSocket} socket${CPUs / coresPerSocket > 1 ? 's' : ''} with ${coresPerSocket} core${coresPerSocket > 1 ? 's' : ''} per socket`}</option>
-            )}
+            <option>{formatMessage(messages.vmChooseCoresPerSocket)} </option>
+            {
+              map(
+                this._getCoresPerSocketPossibilities(),
+                coresPerSocket => <option
```

Comment:
Check your browser's console for warnings.

Everytime `map` is used, a `key` prop should be added to items.